docs/network.md

```diff
@@ -0,0 +1,55 @@
+# 10. Message transportation layers.
+
+This section describes different protocols of sending messages currently used in `Near`
+
+## 10.1 Messages between Actors.
+
+`Near` is build on `Actix`'s `actor` framework. (https://actix.rs/book/actix/sec-2-actor.html)
+Usually each actor runs on its own dedicated thread.
+Only messages implementing `actix::Message`, can be sent using between threads.
```

Comment:
I'd also mention that those actors have their own queues - and that the processing of the requests is happening asynchronously.

Reply:
(but as these are all 'within' one program - we do not need to worry about backwards compatibility.